### PR TITLE
[12.0][MIG] account_invoice_production_lot

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of account_invoice_production_lot module to v12

### CODE REVIEW

**Comment by pedrobaeza** [comment]
It's interesting to review the related PR #99.

FYI, Odoo v13 will include a mechanism for putting already this information in invoice, so hopefully this module won't be needed.

**Comment by pedrobaeza** [comment]
Are you going to fix Travis and runbot?

File: account_invoice_production_lot/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                            _('S/N'), sml.lot_id.name,
+                        ))
+                    else:
+                        lot_strings.append('<li>%s %s (%s)</li>' % (
+                            _('Lot'), sml.lot_id.name, sml.qty_done,
+                        ))
+            if lot_strings:
+                note += ' '.join(lot_strings)
+                note += '</ul>'
+                line.lot_formatted_note = note
```
Comment (sergio-teruel) [inline account_invoice_production_lot/models/account_invoice.py]: Why not group by lots...? I attach a picture. By this way we cosume a lot vertical space
And is posible do this directly in report?

Selección_299

**Comment by pedrobaeza** [comment]
@jbeficent can you attend this?

File: account_invoice_production_lot/models/account_invoice.py (hunk 1 above)
Comment (AdriaGForgeFlow) [reply account_invoice_production_lot/models/account_invoice.py]: Lots should be grouped correctly now

**Comment by sergio-teruel** [comment]
Please fix travis.. 
I think that field lot_formatted_note is not very inheritable..
Mybe you could return a list of dictionaries with lot and units and do render in a report view.
¿What do you think?

**Comment by sergio-teruel** [comment]
PR superseed in https://github.com/OCA/account-invoice-reporting/pull/123

**Pull request closed without merging**